scripts/sanitycheck

```diff
@@ -1164,9 +1164,9 @@ class TestCase:
         self.depends_on = tc_dict["depends_on"]
         self.min_flash = tc_dict["min_flash"]
         self.extra_sections = tc_dict["extra_sections"]
-        self.path = os.path.join(os.path.basename(os.path.abspath(testcase_root)),
-                                 workdir, name)
-        self.name = self.path # for now
+
+        self.path = os.path.normpath(os.path.join(os.path.abspath(testcase_root).replace(ZEPHYR_BASE + "/",''), workdir, name))
+        self.name = os.path.join(self.path)
```

Comment:
what effect does os.path.join() have with just one argument? I'm confused why we need this line

Reply:
when you run with -T <path to test>, the test identifier is shortened and is relative to the root of the testcase path. We are using the path/test_name as the identifier for each test, so this will be seen as another new test.